v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint needs to be made:
+
```

Comment:
need to also add snippets for golang, python and curl (see how these are done in other doc pages - we have reusable components for them)

Reply:
@rishabhpoddar can you please review the JS code snippets so that i can port them to the other languages